README.md

```diff
@@ -370,6 +370,10 @@ Additional configuration can be passed via the following environment variables:
 * ACTOR_TIMEOUT - default `60 seconds`
 * AKKA_HTTP_SERVER_REQUEST_TIMEOUT - `default 60 seconds`
 * AKKA_HTTP_SERVER_IDLE_TIMEOUT - `default 60 seconds`
+* TO_REGISTRY - `default false` reports lag/offset/end to metricsRegistry
```

Comment:
Is there any necessary configuration here to specify the region of the cloudwatch endpoint to use? I had eu-central-1 specified in "~/.aws/config" but it continued to default to us-east-1

Reply:
ah crap...i probably have something in there, ill look into it and document it

Reply:
this seems to work for me, not sure why you are seeing it and I don't see any reference to us-east-1 in the code (which is my default btw)

➜  remora git:(cloudwatch) cat ~/.aws/config
[default]
output = json
region = us-east-1

Reply:
resolved